packages/react-scripts/scripts/eject.js

```diff
@@ -129,7 +129,7 @@ prompt(
   // Add Jest config
   console.log('  Adding ' + cyan('Jest') + ' configuration');
   appPackage.jest = createJestConfig(
-    filePath => path.join('<rootDir>', filePath),
+    filePath => path.posix.join('<rootDir>', filePath),
```

Comment:
To clarify, path.win32 supports `/` and `\` (and outputs `\`) but path.posix only supports `/`.

See this discrete note at the bottom: https://nodejs.org/api/path.html#path_path_win32